apps/web-neutron/packages/ui/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default as chainConfig } from '@/chainConfig';
```

Comment:
I see these files `apps/web-neutron/packages/ui/**` were added in your last commits, was that intentional? Since it is the first chain where we have these files (in other chains there is no `packages` inside `apps/web-*`)

Reply:
updated it now, thanks!